src/cloud-api-adaptor/docs/initdata.md

```diff
@@ -166,5 +166,16 @@ It also calculates the digest `/run/peerpod/initdata.digest` based on the `algor
 
 The digest can be calculated manually and set to attestation service policy before hand if needed. To calculate the digest, use a tool (for example some online sha tools) to calculate the hash value based on the initdata raw string. The calculated sha384 is: `52af3178dd7ad4bf551e629b84b45bfd1fbe1434b980120267181ae3575ea20ca9013b8eadf31d27eed7ff2552d500ef` for above sample.
 
+For example, for ibmse TEE driver, the `se.user_data` can be set as:
```

Comment:
you may want to add a link here to the attestation policy ref in trustee.

Reply:
Thanks @bpradipt , added the claims link in trustee repo here.